src/routes/NarcologerPage/NarcologerPage.css

```diff
@@ -0,0 +1,9 @@
+body {
+    background: #050505;
+  }
```

Comment:
Here is another issue. Because of the use of global style, the background turns black on all pages. Use css class instead for your root element.